### PR TITLE
Propagate input to predict output fn

### PR DESCRIPTION
**Pull request by martinbomio**
This is a very simple way of solving https://github.com/spotify/scio/issues/902.
Tested it with my own workflow and seems to be working.

### CODE REVIEW

File: scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowSCollectionFunctions.scala [hunk 1]
```diff
@@ -44,7 +44,7 @@ private class PredictDoFn[T, V](graphBytes: DistCache[Array[Byte]],
                                 fetchOp: Seq[String],
                                 @Nullable config: Array[Byte],
                                 inFn: T => Map[String, Tensor],
-                                outFn: Map[String, Tensor] => V) extends DoFn[T, V] {
+                                outFn: (T, Map[String, Tensor]) => V) extends DoFn[T, V] {
```
Comment (martinbomio) [inline scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowSCollectionFunctions.scala]: Another option is to create a custom type or case class like `PredictionResult(original: T, outputOps: Map[String, Tensor])`

**Comment by ravwojdyla** [comment]
LGTM, still complex API, but we can address that later. @richwhitjr would this help in your case too?

**Comment by richwhitjr** [comment]
Yep, lets me try the API at least in a real word example.

**Comment by martinbomio** [comment]
@richwhitjr I tried it in one of our workflows, but please try it!

**Comment by ravwojdyla** [comment]
@richwhitjr any update here?

**Comment by richwhitjr** [comment]
Don't have a TF model handy right now.  I can pull an old example and see if it works compared to calling the API directly.

**Comment by richwhitjr** [comment]
Yep this works for me on a small local test.  LGTM

**Pull request merged**